generated/protocol-tests/test-clients/aws-cpp-sdk-rest-xml-protocol/source/model/NestedXmlMapWithXmlNameRequest.cpp

```diff
@@ -37,7 +37,14 @@ Aws::String NestedXmlMapWithXmlNameRequest::SerializePayload() const
      XmlNode nestedXmlMapWithXmlNameMapKeyNode = nestedXmlMapWithXmlNameMapMapEntryNode.CreateChildElement("OuterKey");
      nestedXmlMapWithXmlNameMapKeyNode.SetText(mapItem.first);
      XmlNode nestedXmlMapWithXmlNameMapValueNode = nestedXmlMapWithXmlNameMapMapEntryNode.CreateChildElement("value");
-     nestedXmlMapWithXmlNameMapValueNode.SetText(mapItem.second);
+     for(const auto& nestedXmlMapWithXmlNameMapValueNestedItem : mapItem.second)
```

Comment:
"NestedItem" comes from the codegen to distinguish it from the pre-existing outer loop "item".
"nestedXmlMapWithXmlNameMapValue" comes from the c2j model and we don't control it.

Should real service client use this feature - the variable name should be better.